src/platform/plugins/shared/dashboard/server/content_management/v3/cm_services.ts

```diff
@@ -456,6 +456,11 @@ export const dashboardSearchOptionsSchema = schema.maybe(
       kuery: schema.maybe(schema.string()),
       cursor: schema.maybe(schema.number()),
       limit: schema.maybe(schema.number()),
+      spaces: schema.maybe(
```

Comment:
Can you define the behavior when spaces is not provided? Does it only search the current space?